.pylintrc

```diff
@@ -0,0 +1,579 @@
+[MASTER]
```

Comment:
Was this Pylint config auto-generated, or grabbed somewhere from Pylint?

Reply:
Initially this config was auto-generated and then I added some changes like:
1) add notes to the `disable` in the `[MESSAGES CONTROL]` section
2) add notes to the  `[MISCELLANEOUS]` section
3) add names to `good-names` in the `[BASIC]` section

Reply:
Let's discuss `disable` in `[MESSAGES CONTROL]` and `notes` in `[MISCELLANEOUS]`.

Reply:
Done.